### PR TITLE
Add job publishing `tbtc-v2.ts` package for `goerli`

### PR DESCRIPTION
**Pull request by michalinacienciala**
So far in the CI we've been only publishing the `tbtc-v2.ts` package built based
on `development`-tagged `tbtc-v2` package. Now we add a new job that will update
the `tbtc-v2` dependency to use `goerli`-tagged package and will build and
publish `tbtc-v2.ts` package based on that.

We also bump up version of the package in the `package.json`.

NOTE: Before merging this PR we need to merge https://github.com/keep-network/ci/pull/36.

### CODE REVIEW

File: .github/workflows/typescript.yml [hunk 1]
```diff
@@ -63,6 +74,73 @@ jobs:
...
+
+      - name: Install dependencies
+        run: yarn install --frozen-lockfile
+
+      - name: Get upstream packages versions
+        uses: keep-network/ci/actions/upstream-builds-query@v2
+        id: upstream-builds-query
+        with:
+          upstream-builds: ${{ github.event.inputs.upstream_builds }}
+          query: tbtc-v2-contracts-version = github.com/keep-network/tbtc-v2#version
```
Comment (nkuba) [inline .github/workflows/typescript.yml:110]: It also needs `@keep-network/ecdsa` contracts dependency.

**Review by nkuba — CHANGES_REQUESTED** [review]

File: .github/workflows/typescript.yml (hunk 1 above)
Comment (michalinacienciala) [reply .github/workflows/typescript.yml]: Right, it looks that I created a branch from the code that did not have the `ecdsa` dependency. I updated the code and added the `ecdsa` upgrade in [a7bb07b](https://github.com/keep-network/tbtc-v2/pull/522/commits/a7bb07bf8708535b6505077de91c431aed31a083).

File: .github/workflows/npm-typescript.yml [hunk 2]
```diff
@@ -33,7 +33,10 @@ jobs:
         run: git config --global url."https://".insteadOf git://
 
       - name: Resolve latest contracts
-        run: yarn upgrade @keep-network/tbtc-v2 --exact
+        run: |
+          yarn upgrade \
+            @keep-network/ecdsa
+            @keep-network/tbtc-v2 --exact
```
Comment (nkuba) [inline .github/workflows/npm-typescript.yml:39]: Could we move `--exact`?
```suggestion
          yarn upgrade --exact \
            @keep-network/ecdsa
            @keep-network/tbtc-v2
```

**Review by nkuba — APPROVED** [review]

File: .github/workflows/npm-typescript.yml [hunk 3]
```diff
@@ -33,7 +33,10 @@ jobs:
         run: git config --global url."https://".insteadOf git://
 
       - name: Resolve latest contracts
-        run: yarn upgrade @keep-network/tbtc-v2 --exact
+        run: |
+          yarn upgrade \
+            @keep-network/ecdsa
```
Comment (nkuba) [inline .github/workflows/npm-typescript.yml:38]: We need `\`:
```suggestion
            @keep-network/ecdsa \
```

**Review by nkuba — CHANGES_REQUESTED** [review]

**Review by nkuba — APPROVED** [review]

**Pull request merged**